extensions/wikia/Search/classes/Services/FandomSearchService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Wikia\Search\Services;
+
+class FandomSearchService extends EntitySearchService {
+	const RESULTS_COUNT = 5;
+
+	protected function getCore() {
+		return 'fandom';
+	}
+
+	protected function prepareQuery( $query ) {
+		$select = $this->getSelect();
+
+		$phrase = $this->sanitizeQuery( $query );
+		$select->setQuery( 'title:' . $phrase . ' excerpt_t:' . $phrase . ' content_t:' . $phrase );
+		$select->clearFields()->addFields( [ 'title', 'url', 'image_s', 'excerpt_t', 'vertical_s' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)